src/Bicep.Cli/Services/CompilationService.cs

```diff
@@ -104,8 +109,27 @@ public ParamsSemanticModel CompileParams(string inputPath, bool skipRestore)
             {
                 throw new Exception($"Unable to read file {inputPath}");
             }
+            
+            var paramsFile = SourceFileFactory.CreateBicepParamFile(inputUri, fileText);
+
+            var usingDeclarations = paramsFile.ProgramSyntax.Children.OfType<UsingDeclarationSyntax>();
+
+            if(!PathHelper.TryGetUsingPath(usingDeclarations.SingleOrDefault(), out var bicepFilePath, out var failureBuilder))
```

Comment:
This is more of a design question for a future PR, but should an exception be thrown when a parameter file doesn't have a `using` statement? Type checking and completions are only available with a linked template, but there's no reason a parameter file can't be compiled to ARM independent of any template.